config/initializers/1_fast_gettext.rb

```diff
@@ -15,6 +15,11 @@
 # work in all domains context by default (for plugins)
 include FastGettext::TranslationMultidomain
 
+# Keep TRANSLATORS comments
+Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=TRANSLATORS:]
+# Disable fuzzy .po merging
+Rails.application.config.gettext_i18n_rails.msgmerge = %w[--no-fuzzy-matching]
```

Comment:
>  Can you provide the specifics of what string and translation is in each file as you extract, pull and merge?

What do you need? There aren't any translations yet for REX.

I followed the wiki page, incorporated the changes from https://github.com/theforeman/foreman_plugin_template/pull/13, and rake the rake task.  I'm not sure what I need to do to verify that things worked as expected though.